2025/lab/Keys/Week11_Ordinal_Multinomial.Rmd

```diff
@@ -0,0 +1,494 @@
+      1. Women gaining the right to vote
+      2. The availability of the birth control pill
+      3. Passage of the Equal Pay Act
+      4. Passage of the Family and Medical Leave Act "
+
+
+# Data Loading and Management
+
+The dataset is in the lab data folder. Let's load it up and do some data management. 
+
+```{r gss-loading}
+pewraw <- haven::read_sav("../DATASETS/Mar20_19th_Amendment_cleaned dataset.sav")
+
+pewamend <- pewraw %>% 
+  dplyr::select(
+    
+    # Target Vars
+    EQRIGHTS2, ADVANCE3,
+    
+    # IV's
+    age = ppage, 
+    educ = ppeducat,
+    race = ppethm,
+    gender = ppgender,
+    marital = ppmarit,
+    employ = ppwork,
+    party = PARTY,
+    ideo = IDEO,
+    weight, 
+    ) %>% 
+  mutate(
+    
+    # OUTCOME VARIABLES
+    rightsprog = ordered(EQRIGHTS2,
+                        levels = c(1, 3, 2),
+                        labels = c("Too Far", "About Right","Not Far Enough")),
+    milestone = factor(ADVANCE3,
+                       levels = c(1:4),
+                       labels = c("Voting", "BC_Pill", "EQ_Pay", "FMLA")),
+    # PREDICTORS
+    gender = ifelse(gender==1, "Male", "Female"),
+    race = case_when(
+      race == 1 ~ "White",
+      race == 2 ~ "Black", 
+      race == 3 ~ "Other", 
+      race == 4 ~ "Hispanic",
+      race == 5 ~ "Mixed"
+      ),
+    educ = factor(educ, levels = c(1:4),
+                  labels = c("<HS", "High School", "Some College", "College+")),
+    marital = factor(marital, 1:6,
+                     labels = c("Married", "Cohabiting","Divorced",
+                                "Separated","Widowed","Never Married")),
+    employ = case_when(
+      employ == 1  ~ "Working",
+      employ ==  2 ~ "Self-Employed",
+      employ %in% c(3,4) ~ "Unemployed", # Looking for  work or temporary layoff
+      employ %in% c(5,6,7) ~ "Not Working"   # Retired, disabled,  other
+      ),
+    party = case_when(
+      party == 1 ~ "Republican",
+      party == 2 ~ "Democrat",
+      party == 3 ~ "Independent",
+      party == 4 ~ "Something Else"
+      ),
+    ideo = factor(ideo, levels = c(1:5),
+                  labels = c("Very Conservative", "Conservative", "Moderate", "Liberal", "Very Liberal")),
+    weight = as.numeric(weight)
+  ) %>% 
+  dplyr::select(-EQRIGHTS2, -ADVANCE3)
+
+```
+
+If you're curious, the code to view the questions and response orders is "lab_data/checking_atp_coding.R". It uses the `foreign::read.spss` function to maintain variable "attributes." This allows for maintaining data information but makes recoding variables slightly harder. Instructions for how to do this are also in the file "lab-data/atp_codebook.pdf".
+
+## Missingness Analysis
+
+Since we're working with new data, let's see if there are any patterns of missingness among the variables. 
+```{r miss-upset}
+gg_miss_upset(pewamend)
+```
+
+Seeing no particular pattern, let's employ listwise deletion. 
```

Comment:
This is fine, but I'd just emphasize that this would need more serious consideration in a real research setting. The comment about "no particular pattern" comes across as too hasty an evaluation.